static/src/javascripts/projects/admin/modules/abtests/abtest-report-item.js

```diff
@@ -10,7 +10,7 @@ import bonzo from 'bonzo';
 import debounce from 'lodash/functions/debounce';
 
 class ABTestReportItem extends Component {
-    constructor(config: any): void {
+    constructor(config: Object): void {
```

Comment:
Interesting, I've not actually seen an empty object literal used as the type placeholder yet, only object literals where the expected properties were also defined. I'll look into the difference.